config/routes.rb

```diff
@@ -3,6 +3,10 @@
 
   root "home#show"
 
+  get '/auth/twitter', as: :login
```

Comment:
Just so I understand. When the user is directed to this route, the oauth process starts automatically, and the callback is directed to the sessions controller where the user and session is created?